edgelet/iotedge/src/config/apply.rs

```diff
@@ -406,6 +406,8 @@ fn execute_inner(
 
             homedir: AZIOT_EDGED_HOMEDIR_PATH.into(),
 
+            allow_privileged_docker_containers: true,
```

Comment:
We're not going to have a super-config flag for it?

Reply:
ah, I misread what this file was doing. I thought it was setting default values, not pulling from the super_config. I'll add it to the super-config.

Reply:
You also need to add a commented-out blurb to [`template.toml`](https://github.com/Azure/iotedge/blob/master/edgelet/contrib/config/linux/template.toml)

Reply:
Done